projects/plugins/jetpack/extensions/blocks/sharing-button/variations.js

```diff
@@ -89,6 +89,14 @@ export const variations = [
 		title: 'Nextdoor',
 		icon: <SocialIcon icon={ 'nextdoor' } size={ 24 } />,
 	},
+	{
+		name: 'share',
+		attributes: { service: 'share', label: 'Share' },
+		title: 'Native Share',
```

Comment:
I'm thinking that since this is not a brand name, we may want it to be translatable?